packages/libs/eda/src/lib/core/components/visualizations/implementations/HistogramVisualization.tsx

```diff
@@ -560,13 +552,6 @@ function HistogramViz(props: VisualizationProps<Options>) {
       )
         return undefined;
 
-      // wait till distributionDataPromise is ready
```

Comment:
There was no need to wait for this - it was indirectly causing a double data request due to `distributionDataPromise.*` being added as dependencies.

Reply:
Though waiting for it probably avoided the runtime bug that I later [fixed](https://github.com/VEuPathDB/web-monorepo/pull/821/commits/87b7ae9f5875372b093ddd1e2c4cba8054110bbb).